requirements/test-acceptance.txt

```diff
@@ -1,2 +1,2 @@
-bok_choy==0.3.1
-nose==1.3.0
+-e git+https://github.com/edx/bok-choy.git@4a259e3548a19e41cc39433caf68ea58d10a27ba#egg=bok_choy
```

Comment:
I think this does not need to be this exact. The latest version 0.3.2 would do fine.